### DISCUSSION TITLE
ChoiceGroup theme help

### DISCUSSION THREAD

**Discussion by OliverFlint** [microsoft/fluentui#20007] [Q&A]
Hi, Does anyone have an example of the choicegroup control working with a theme?

Although I apply a theme it's not rendering as expected.

Example: https://codepen.io/OliverFlint/pen/WNOLPKv
I'm expecting this example to be black and white and not blue.

Thanks in advance!

**Comment by OliverFlint** [comment]
I have solved my own "problem" but it's not using themes... https://codepen.io/OliverFlint/pen/KKqbLwO

**Reply by micahgodbolt** [reply]
that's a perfectly fine way to do it if you are trying to target that one component specifically.
